docs/pages/install.md

```diff
@@ -0,0 +1,110 @@
+
+###### APK (Alpine Package Keeper):
+You want to install a `.apk` package if you are using an Alpine Linux system.
+
+###### DEB (Debian Package):
+You want to install a `.deb` package if you are using a Linux distribution system such as: Debian, Ubuntu, or something similar.
+
+##### What about tar and zip?
+If you want a more manual setup, you can download the `.tar.gz` or `.zip` files and extract the binary where you need it. However, the above packages are recommended for more inexperienced users.
+
+##### What version should I download?
+Our versions are built like so:
+For example: 7.1.2
+- 7 represents the latest major version release
+- 1 represents feature releases
+- 2 represents a bug fix/patch release
+
+We recommend you to use the latest feature or major release version so it includes the latest bug fixes and features.
+
+### 2. Extract the Binary
+Once the package has finished downloading, place it in your workspace
+
+##### RPM:
+To install RPM packages, you can use the dnf or yum package manager. Just make sure you have sudo access if you are not root:
+```console
+sudo yum install pelican-7.1.2-1.x86_64.rpm
+```
+Replace 'yum' with 'dnf' for dnf install
+
+##### DEB:
+To install Debian packages, you can use apt or dpkg package manager:
+```console
+sudo dpkg -i pelican-7.1.2-1_amd64.deb
+sudo apt-get install -f
+```
+Or
+```console
+sudo apt install ./pelican-7.1.2-1_amd64.deb
+```
+
+##### APK:
+To install APK packages on Alpine Linux, you can use the apk package manager:
+```console
+apk add pelican_amd64.apk
+```
+
+##### Tarballs:
+To install `.tar.gz` packages, you can extract with tar:
+```console
+tar -xzf pelican_Darwin_x86_64.tar.gz
+```
+
+##### Zip:
+To install `.zip` packages, you can use [7zip](https://www.7-zip.org/) or other programs to unzip your file. Simply right click on the `.zip` file and extract all contents to a directory of your choice 
+
+### 3. Test Functionality of the Pelican Platform
+##### For rpm, deb, and apk
+If you downloaded this way, Pelican should automatically be added to the path. You can check this by:
+```console
+which pelican
+```
+and make sure you get an output. You can then check functionality by running:
+```console
+pelican -f osg-htc.org object copy /osgconnect/public/osg/testfile.txt .
+testfile.txt 36.00 b / 36.00 b [=============================================================================================] Done!
+```
+
+##### For tarballs and zip, using the binary itself:
+Once extracted, make sure you are in the same directory as the **Pelican** executable. To test if everything works, we can do a simple **object copy** command:
+
+```console
+./pelican -f osg-htc.org object copy /osgconnect/public/osg/testfile.txt .
+testfile.txt 36.00 b / 36.00 b [=============================================================================================] Done!
+```
+**Note:** If you are running on windows, you will need to add '.exe' to './pelican' to be './pelican.exe ...'
```

Comment:
Is this true? When I downloaded and extracted the tarball for Windows, the executable is already called `pelican.exe`.

Reply:
yup, you're right. Thanks for catching that